frontend/src/pages/Home.jsx

```diff
@@ -7,7 +8,8 @@ const Home = ({navigate}) => {
         <div>
             {/* <h1>Home Page:</h1>
             <h1>Bald Bible</h1> */}
-            <h2 id="home-statement">Social network for people who are bald or experiencing hair loss</h2>
+            <h2 className='mainHeader' id="home-statement">Welcome to Bald Bible</h2>
+            <p className='mainTagline' id="home-statement">A positive corner of the internet to help you accept your hair loss . . . Not fight it.</p>
```

Comment:
Purely personal opinion here, 'accept' feels like resignation and has a sense of reluctance. How about 'embrace'? Otherwise, I really like the change. Despite the positive reason, I also wonder if the '...not fight it' introduces negativity and maybe it feels more positive as a singular positive statement. Your call.